src/App.js

```diff
@@ -1,16 +1,42 @@
 import React, { Component } from 'react'
 import { Instructions } from './components'
+import { Counter } from './components'
 import './styles/app.css'
 
 class App extends Component {
+  //  var mylist = ["Ayy", "whats", "good"]
+  //  const mapper = list.map(x : x + "\n")
   render() {
+    let mylist = ['Ayy', 'whats', 'good']
+    mylist = mylist.filter(x => x.length > 2)
+    mylist = mylist.map((x, index) => (index % 2 === 1 ? x.toUpperCase() : x))
+
+    const mapper = mylist.map(x => (
+      <div>
+        <ul>
+          <li>{x}</li>
+        </ul>
+      </div>
+    ))
```

Comment:
make sure you are setting the `key` prop when rendering lists: https://egghead.io/lessons/egghead-use-the-key-prop-when-rendering-a-list-with-react